docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst

```diff
@@ -415,18 +415,14 @@ mark a conversation session, as shown in the samples below:
 
 
       For more information, refer to the
-      `C++ sample <https://github.com/openvinotoolkit/openvino.genai/blob/master/samples/cpp/chat_sample/README.md>`__
+      `C++ sample <https://github.com/openvinotoolkit/openvino.genai/tree/master/samples/cpp/chat_sample/>`__
 
 
 Using GenAI with Vision Language Models
 #######################################
 
-OpenVINO GenAI introduces the ``openvino_genai.VLMPipeline`` pipeline for
-inference of multimodal text-generation Vision Language Models (VLMs) such as MiniCPM-V series.
-miniCPM-V-2_6, as well as other models from the series, are Large Language Models that
-support multilingual multimodal interaction.
-
-A chat scenario is presented in the samples below:
+With a text prompt and image as input, VLMPipeline can generate text using
```

Comment:
I think we can  keep this first sentence to give the perspective that this pipeline is multimodal. 
OpenVINO GenAI introduces the ``openvino_genai.VLMPipeline`` pipeline for
inference of multimodal text-generation Vision Language Models (VLMs)